fakturogen-user/src/main/java/pl/fakturogen/user/web/dto/UserDTO.java

```diff
@@ -1,4 +1,18 @@
 package pl.fakturogen.user.web.dto;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
 public class UserDTO {
+    private Long id;
+    private String email;
+    private String firstname;
+    private String lastname;
+
 }
```

Comment:
Probably you gonna have conflict with this class. Please use mine to resolve it.